### DISCUSSION TITLE
How to temporarily disable proxy?

### DISCUSSION THREAD

**Discussion by pedoc** [ScoopInstaller/Scoop#6111] [Help and Questions]
I have configured a global proxy via scoop config proxy ``127.0.0.1:10808``.
Now I want to temporarily stop using the proxy. 
In addition to clearing the proxy settings, is it also possible to temporarily disable the proxy via parameters? 
For example, similar to the ``--no-proxy`` parameter of ``winget-cli``

thanks

**Comment by niheaven** [comment]
At present, the only method to temporarily disable a proxy is to remove it entirely.

**Comment by Ra2-IFV** [comment]
You may wish to use aria2 together with environment variables.
Why not write a script for this? just `scoop config proxy xxx` and `scoop config proxy ''`
